### PR TITLE
Sr 487 download text

### PR DESCRIPTION
**Pull request by Sveouu**
includes:
 - updates to 488 household histogram (vertical annotation outstanding)
 - 487 sample download captions with option to copy in plain text & bibText included in the download component.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -13,5 +13,6 @@
   },
   "[json]": {
     "editor.defaultFormatter": "esbenp.prettier-vscode"
-  }
+  },
+  "python.pythonPath": "C:\\Python27\\ArcGIS10.7\\python.exe"
```
Comment (Fuhji) [inline .vscode/settings.json:17]: I dont think this is needed in this repo

File: src/app/pages/quickMaps/pages/baselineDetails/householdSupply/householdSupply.component.html [hunk 2]
```diff
@@ -2,7 +2,9 @@
   <mat-tab-group mat-align-tabs="center" [(selectedIndex)]="selectedTab">
     <mat-tab label="Household Histogram">
       <ngx-chartjs *ngIf="null != chartData" id="histo" [data]="chartData.data" [type]="chartData.type"
-        [options]="chartData.options" [plugins]="chartData.plugins"></ngx-chartjs>
+        [options]="chartData.options" [plugins]="chartData.plugins">
+
```
Comment (Fuhji) [inline src/app/pages/quickMaps/pages/baselineDetails/householdSupply/householdSupply.component.html:6]: wonder why VSCode isnt removing the empty line spacing here

File: .vscode/settings.json (hunk 1 above)
Comment (Sveouu) [reply .vscode/settings.json:17]: yep, not sure how it got there tbh.

**Review by Fuhji — CHANGES_REQUESTED** [review]
Just need to remove the python from settings.json otherwise all good!

**Review by bgsandan — APPROVED** [review]
Other than @Fuhji comments, looks good to me

**Review by Fuhji — APPROVED** [review]

**Pull request merged**